src/obsidian_ai_hub/web/routes/coding.py

```
@@ -0,0 +1,176 @@
+    result = []
+    for proj in all_projects:
+        project_path = proj.get("project_path")
+        is_valid = False
+        repo_path = None
+        error_msg = None
+
+        if project_path:
+            try:
+                repo_path = backend.validate_git_repo(project_path)
+                is_valid = True
+            except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)